cypress/integration/docs-links/docs-links.spec.ts

```diff
@@ -1,4 +1,4 @@
-import docsUrls from "@maas-ui/maas-ui/src/app/base/docsUrls";
+import docsUrls from "../../../src/app/base/docsUrls";
```

Comment:
driveby fix for docs URLs check - we can set up an import resolver in a follow up so for now it's just a relative import